examples/spec/lcdproc/LCDd.ini

```diff
@@ -0,0 +1,2752 @@
+[]
+mountpoint = LCDd.conf
+;infos/plugins = dump validation enum range ipaddr path network type
+infos/plugins = dump validation enum
+
+;Until the max plugin error is not fixed, some specifications will be left commented out
+
+;=============================================================================
+;=============================================================================
+
+;Affecting all subsettings, eg curses/Contrast, CFontz/Contrast, etc.
+;------------------------------------
+
+[/_/contrast]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+description = Set the initial contrast
+
+[/_/brightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/offbrightness]
+type = unsigned_short
+check/type = unsigned_short
+check/range = 0-1000
+
+[/_/reboot]
+type = boolean
+check/type = boolean
+default = no
```

Comment:
Needs boolean plugin?

Reply:
It does not load the boolean plugin? I want the type plugin solely:

> The type checker plugin supports all basic CORBA types: ... , boolean, ... When checking any it will always be successful, regardless of the content.

Reply:
Without the boolean plugin, only "1" and "0" are valid.

Reply:
You are right, still I cannot load the boolean plugin though (#2451)

Reply:
When looking into the plugin it does not state that "no/yes" are actually valid options.
Only `on/true`, `on/false`, `on/invalid`, `on/invalid/warning,`true`,`false`.
So I have to change the yes/no to enums I think

Reply:
You can also introduce yes/no as valid values in the boolean plugin. It seems like these values make sense, too.

Reply:
@kodebach: I think we need to change it to enum rather than boolean because the boolean plugin would convert it to 0 and 1 and LCDproc checks for "yes/no", "on/off". Is this correct?

Reply:
No, definitely don't convert it to enum. If it is a boolean than declare it as a boolean. The old LCDproc does accept yes/no, on/off and a few other things for booleans but that should handled by Elektra and LCDproc should only expect 0 and 1 for booleans.